lexer_test.go

```diff
@@ -371,10 +383,96 @@ func TestLexer_Find(t *testing.T) {
 	})
 }
 
+func TestLexer_Ignore(t *testing.T) {
+	t.Parallel()
+
+	t.Run("basic", func(t *testing.T) {
+		t.Parallel()
+
+		l := NewLexer(runeio.NewReader(strings.NewReader("Hello\n!Ignore!\n")), &wordState{})
+
+		advanced, err := l.Advance(7)
+		if err != nil {
+			t.Errorf("unexpected error: %v", err)
```

Comment:
nit: This should be `t.Fatalf`

Reply:
Should I push a change to this branch or just make the change part of a later PR?

Reply:
Your call. Usually these kind of changes are easy enough to do that you can do it on that PR but I'll just go ahead and merge this one so it's not held up. Feel free to submit a second PR if you have the bandwidth

Reply:
I'll merge this first and change those in a later PR.